Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -385,7 +385,7 @@ try {
         -installApps $installApps `
         -installTestApps $installTestApps `
         -installOnlyReferencedApps:$settings.installOnlyReferencedApps `
-        -generateDependencyArtifact:$settings.generateDependencyArtifact `
+        -generateDependencyArtifact `
```

Comment:
Why is `$settings.generateDependencyArtifact` not used here?
Is it used anywhere, as I see it is being retrieved in the `ReadSettings` action.

Reply:
Because Run-AlPipeline should now always generate dependency artifact 
The setting is now retrieved as an env variable in order to only publish artifacts if the setting is set.
The change in this PR is to publish a thisbuild artifact with this dependency artifact.